connect/src/state.rs

```diff
@@ -87,7 +87,7 @@ pub struct ConnectConfig {
     pub initial_volume: u16,
     /// Disables the option to control the volume remotely (default: false)
     pub disable_volume: bool,
-    /// The steps in which the volume is incremented (default: 1024)
+    /// The steps in which the volume is incremented (default: 64)
```

Comment:
Ah dang it, I did overlooked the doc-comment description. Could you update it, so that it states that it is the amount of steps and not the size of a step?

> You could probably also just copy-paste the description of the new parameter :D